spec/system/ndbn_members_system_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe "NDBN Members Upload", type: :system do
+  # TODO: This test crashes right after the upload
+  xit "CSV upload flow functions without error" do
+    user = create(:super_admin)
+    sign_in(user)
+    visit admin_ndbn_members_path
+
+    attach_file("member_file", "spec/fixtures/ndbn-large-import.csv")
+    click_button "Upload"
+
+    expect(page).to have_content("NDBN Members have been updated!")
+  end
+end
```

Comment:
Not sure how to address this. Ideally I would like a system test just to run the full flow. In theory this should work but when you upload the file it gets attached and then the browser crashes.

Reply:
Nice, it worked.